tools/Customization/DevHome.Customization/Views/FileExplorerPage.xaml

```diff
@@ -25,9 +34,29 @@
                   MinWidth="300" 
                   HorizontalAlignment="Left"/>
 
-                    <Button  Content="Add" 
+                    <Button  
+                         Content="Add" 
                          Click="AddButton_Click" 
-                         Margin="10, 27, 0, 0"/>
+                         Margin="10, 27, 0, 0">
+                    <Button.Resources>
+                        <Flyout x:Name="LocalRepositoryProvidersFlyout" Placement="Bottom">
+                                <ItemsRepeater ItemsSource="{x:Bind ViewModel.LocalRepositoryProviders}"
```

Comment:
Why an ItemsRepeater?  Why won't a ListView, or another defined control work here?

Reply:
As this UI is temporary, I will be sure to factor this feedback in a future PR if applicable to the actual UI design. Thank you!